cmd/ocm-backplane/login/login.go

```diff
@@ -15,41 +15,42 @@ import (
 	"k8s.io/client-go/tools/clientcmd/api"
 
 	BackplaneApi "github.com/openshift/backplane-api/pkg/client"
+	"github.com/openshift/backplane-cli/pkg/cli/config"
+	"github.com/openshift/backplane-cli/pkg/cli/globalflags"
 	"github.com/openshift/backplane-cli/pkg/utils"
 )
 
+// Environment variable that for setting PS1
+const EnvPs1 = "KUBE_PS1_CLUSTER_FUNCTION"
+
 var (
 	args struct {
-		backplaneURL string
-		manager      bool
+		manager bool
 	}
-)
 
-// Environment variable that for setting PS1
-const EnvPs1 = "KUBE_PS1_CLUSTER_FUNCTION"
-
-// LoginCmd represents the login command
-var LoginCmd = &cobra.Command{
-	Use:   "login <CLUSTERID|EXTERNAL_ID|CLUSTER_NAME|CLUSTER_NAME_SEARCH>",
-	Short: "Login to a target cluster",
-	Long: `Running login command will send a request to backplane api
-	using OCM token. The backplane api will return a proxy url for 
-	target cluster. The url will be written to kubeconfig, so we can 
-	run oc command later to operate the target cluster.`,
-	Example:      " backplane login <id>\n backplane login %test%\n backplane login <external_id>",
-	Args:         cobra.ExactArgs(1),
-	RunE:         runLogin,
-	SilenceUsage: true,
-}
+	globalOpts = &globalflags.GlobalOptions{}
```

Comment:
We can perhaps leverage Cobra's `rootCmd.PersistentFlags()` to set the global options in a better way.

Ref: https://github.com/spf13/cobra/blob/main/user_guide.md#persistent-flags

Reply:
Done